excercise-e-shop/assets/scss/page/home/_subcribe.scss

```diff
@@ -0,0 +1,50 @@
+.subcribe {
+  @extend %flex-space-between;
+  background: linear-gradient(90deg, rgba(0,0,0,0.7) 33.28%, rgba(0, 0, 0, 0) 94.05%), url("../img/footer.png");
+  width: 100%;
+  height: 241px;
+  position: relative;
+  .container {
+    @extend %flex-space-between;
+  }
+  .subscribe-title {
+    max-width: 493px;
+    font-size: $md-text-size;
+    color: $secondary-text-color;
+  }
+  .form-subscribe {
+    .email {
+      padding: 14px 23px;
+      border-radius: 61px;
+      width: 351px;
+      margin-right: 40px;
+      font-size:$sm-text-size;
+      color: #ABABAB;
+    }
+  }
+}
+@media only screen and (max-width: map-get($grid-breakpoints, sm )){
+  .subcribe {
+    height: 217px;
+    background: linear-gradient(90deg, rgba(0,0,0,0.7) 33.28%, rgba(0, 0, 0, 0) 94.05%), url("../img/footer-sm.png");
+    .container {
+      flex-direction: column;
+      align-items: flex-start;
+      justify-content: space-evenly;
+      height: 100%;
+    }
+    .subscribe-title {
+      max-width: 200px;
+      font-size: $base-text-size;
+    }
+    .form-subscribe {
```

Comment:
in the same section, `.subcribe-title` and `.form-subcribe` are seemed that not make sense